### DISCUSSION TITLE
grafana-toolkit plugin:dev --watch

### DISCUSSION THREAD

**Discussion by gun88** [grafana/grafana#37270] [General]
Does exist an example that show ho to use the command `"watch": "grafana-toolkit plugin:dev --watch"`?

**Comment by pkolyvas** [comment]
Hey there!

Thanks for taking the time to write us. The best place to ask about workflow question and solicit input/feedback is [on our community forums.](https://community.grafana.com/c/support/33)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
